### PR TITLE
Unnest sources from the framework/src setup

### PR DESCRIPTION
**Pull request by dwijnand**
Part 1 of 2 (hopefully!) of #8992.

### CODE REVIEW

File: documentation/project/plugins.sbt [hunk 1]
```diff
@@ -17,4 +17,4 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-play-enhancer" % "1.1.0")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.0.0")
 
 // Required for Tutorial
-addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "1.4.0-M2")
\ No newline at end of file
+addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "1.4.0-M2")
```
Comment (mkurz) [inline documentation/project/plugins.sbt]: We should use twirl 1.4 final here someday.

Comment (dwijnand) [reply documentation/project/plugins.sbt]: Good eye.  Tracking in https://github.com/playframework/playframework/issues/9024.

**Comment by marcospereira** [comment]
Backport here: #9027.

**Comment by dwijnand** [comment]
Thanks Marcos!

I wish this hadn't been squash merged though...

**Pull request merged**